client/src/RelatedItemsComparison/components/ProductCard/ActionButton.jsx

```diff
@@ -4,6 +4,7 @@ import Icons from '../../../components/Icons';
 
 export default function ActionButton({
   id,
```

Comment:
from a readability POV, maybe clarify what id should be inputted here? Product, review, answer, questions etc